python/ray/data/preprocessors/batch_mapper.py

```diff
@@ -68,6 +76,7 @@ def __init__(
             ],
         ],
         batch_format: Optional[str] = None,
+        batch_size: int = DEFAULT_BATCH_SIZE,
```

Comment:
Do we want to declare `batch_size: Optional[int]`, so user have flexbility to do `batch_size=None` to consume full block as one batch?

I guess it's not a big deal for now, but want to avoid future change for public APIs when we find more edge cases to support.

Reply:
yes, good point!